### PR TITLE
Added support for python list type of BaseModel in response validator

### LINKED ISSUE #55: Support Python types in Response
Hello,
I started using this library recently and notice that I can't use Python types in Response.

Example:

`@api.validate(query=PropertyQueryModel, resp=Response(HTTP_200=list[PropertyModel]), tags=['properties'])`

This raises an Assertion Error from Response __init__
`assert issubclass(value, BaseModel), "invalid `pydantic.BaseModel`"`

I think could be a good improvement to accept Python types, makes sense?
Quickly looking at the code I didn't find other points where it could affect.

Maybe I can help in this part.
Best.

### PR DESCRIPTION
**Pull request by PeterNociar**
I had a go on  adding support for a python type list[BaseModel]  for the Response 

Closes #55

### CODE REVIEW

File: flask_pydantic_spec/types.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import re
-from typing import Optional, Type, Iterable, Mapping, Any, Dict
+from collections import namedtuple
+from typing import Optional, Type, Iterable, Mapping, Any, Dict, is_typeddict, NamedTuple
```
Comment (cgearing) [inline flask_pydantic_spec/types.py:3]: I think `is_typeddict` is only available for 3.10 on - `typing_extensions` should give access to this though - feel free to add it to the requirements and then import from that instead?

Comment (PeterNociar) [reply flask_pydantic_spec/types.py:3]: Ah. Sure I'll fix that

Comment (PeterNociar) [reply flask_pydantic_spec/types.py]: Fixed.

**Comment by robyoung** [comment]
Merged with #62

**Pull request closed without merging**